### DISCUSSION TITLE
Is there supposed to be some sort of time-travel in the game?

### DISCUSSION THREAD

**Discussion by SniperGoth** [redeclipse/base#1167] [General]
Taking a look at all of the game's maps, since the beginning, there was a huge variety of themes, ranging from the medieval  (Castle, Blink, Dawn, Stone, etc ) to the more industrial ones (Industrial, Isolation, Facility, Warp, Forge, etc etc ), to the modern (Rooftop, Fortitude, Canyon, etc), and finally, futuristic/alien looking (Cutec, Ubik, Condensation, Affluence, Oneiroi, etc ).

We know this was done so people could design anything that came from their mind, so their creativity could flow freely, but, in theory, could this mean, in-lore (if there is one in the future), that there is an actual time-travel element that brings the characters/announcer into this variety of arenas and time periods?

**Comment by lambda-11235** [comment]
Or it's just interplanetary travel to civilizations at different points of development.

**Comment by superuser-does** [comment]
Years ago, @qreeves described the lore briefly as Red Eclipse being a bloodsport within a futuristic world, and the TV mode and all that jazz being part of how you spectate.

My own headcanon fits with this, but I'll take it a step further and say it's either aliens or advanced humans and they're cashing in on culture, like Ancient Egypt or medieval castles. A lot of videogames, movies and TV shows today do this, skewing history as necessary for the narrative to work.

So for my part, I'd think of them all as really advanced film sets, which have all been constructed specifically for the game and each has a specific theme. Which is the one closest to how the real world actually is though? Hard to say! Maybe it's a Ready Player One type of situation. Maybe we don't have to explain that part. :)

**Comment by qreeves** [comment]
The best story is the one you make up in your head. You're a tank with a jetpack, shoot stuff, the end.
